### PR TITLE
bj user profile

### PR DESCRIPTION
**Pull request by borvux**

### CODE REVIEW

**Review by reherr — APPROVED** [review]
everything looks good to me!

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:23]: If you ever have an empty image tag, you can either delete it or add default image if the user has not added one, same with the alt tag just for accessibility reasons :)

File: config/routes.rb [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
   devise_for :users  
 
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:6]: great job!

**Review by JoseMDO — APPROVED** [review]
Great job Benny!

**Pull request merged**